assets/js/base/components/product-list/style.scss

```diff
@@ -55,6 +55,11 @@
 	list-style: none;
 }
 
+// The block widgets editor's generic @reset overrides box sizing on all children
+.blocks-widgets-container .wc-block-grid__product {
```

Comment:
Would the fix work if we dropped `.blocks-widgets-container` from the selector? In general, I think we should try to have exactly the same CSS in the frontend and the editor (block editor, site editor or widget editor). Otherwise, things get a bit tricky for themes because there are selectors with different specificity in frontend _vs._ editor so they need to write different styles for each environment.

Reply:
I fully agree that we should aim for having the same CSS for both the frontend and editor. In this case, the issue is that Gutenberg has block widgets editor-specific styles that override the box-sizing of `.wc-block-grid__product`. If we were to drop the class, it'd be the same as the selector that currently gets overridden by the new block widgets editor styles. This is why we, unfortunately, need the extra specificity here.